mock/py/mockbuild/package_manager.py

```diff
@@ -91,12 +91,29 @@ def package_manager_class_fallback(desired, config_opts, bootstrap):
     raise Exception("No package from {} found".format(fallbacks[desired]))
 
 
+def package_manager_class_fallback_from_image(desired, bootstrap):
+    if desired not in fallbacks:
+        raise RuntimeError(f'Unexpected package manager "{desired}"')
+    path = bootstrap.make_chroot_path()
+    for manager in fallbacks[desired]:
+        binary = os.path.join(path, "usr", "bin", manager)
```

Comment:
??? Why not self.config[f'{pm}_command'] ?  I know that we have no config nor self here, but that can be done.

Reply:
This is the `package_manager_class_fallback_from_image()` method - we try to find a pre-existing package manager (any) to install the target package manager.  This is to mimic what we do *on host* when installing bootstrap.  I'll dump there a docstring.